q2_feature_table/tests/test_summarize.py

```diff
@@ -516,5 +490,77 @@ def test_vega_spec_nandling(self):
         self.assertEqual(spec['data'][0]['values'], exp)
 
 
+class TabulateSampleFrequencyTests(TestCase):
+
+    def test_basic_case(self):
+        table = biom.Table(np.array([[25, 25, 25], [25, 25, 25]]),
```

Comment:
Can you make this table more variable? If the underlying code always returned 50 and 2, this test would pass. Make some samples have a different total frequency, and make some features present in only some of the samples.